pyproject.toml

```diff
@@ -65,10 +65,13 @@ select = [
   "E",   # pycodestyle
   "W",   # pycodestyle
   "F",   # pyflakes
+  # "D",   # pydocstyle. TODO: ORQSDK-965
```

Comment:
**question**: I don't see `pydocstyle` in our deps requirements. Is it just a configuration name for `ruff`?

Reply:
It looks like ruff reimplemented the existing pydocstyle tool's checking